README.md

```diff
@@ -179,7 +179,7 @@ header:
     # ignore anything less than this many points from the top, default and per-page
   default: 130
 footer:
-    # ignore anything less than this many points from the bottom, default and per-page
+    # ignore anything more than this many points from the bottom, default and per-page
```

Comment:
looking at [the linked issue](https://github.com/measuresforjustice/textricator/issues/32), I think the text here is not quite on. Maybe:

```
# ignore anything more than this many points from the top, default and per-page
```

(points from top, not points from bottom)